iris_grib/tests/unit/save_rules/__init__.py

```diff
@@ -1,4 +1,4 @@
-# (C) British Crown Copyright 2013 - 2015, Met Office
+# (C) British Crown Copyright 2013 - 2016, Met Office
```

Comment:
The git history has been pulled from `iris` into `iris_grib` so I've honoured the dates here and everywhere else ... just simply updated to this year.